sktree/tree/_unsup_criterion.pxd

```diff
@@ -39,8 +45,12 @@ cdef class UnsupervisedCriterion(BaseCriterion):
     # Unsupervised criterion can be used with splitter and tree methods.
     cdef int init(
         self,
-        const DTYPE_t[:,:] X,
```

Comment:
init no longer takes in X

Reply:
Is this a question, or saying that it's an error?

Reply:
I didn't think my comments would ding you when reviews not started! I noted here b/c I think this here was important